implementing-pagination-and-infinite-scrolling-in-nextjs/index.md

```diff
@@ -0,0 +1,509 @@
+| Search engine optimization (SEO) | Pagination is great for SEO, particularly server-side pagination where the content is fetched and rendered on the server before being sent to the client. This is because search engines are not great at JavaScript-rendered sites.                                       | Infinite scrolling is not good for SEO mainly because search engine crawlers can not yet perfectly emulate manual user behaviors like scrolling which is essential for infinite scrolling, so commonly, most of the site content won’t be accessed which affects the site ranking.                                                                        |
+| User engagement                  | Pagination is better for enabling users to quickly find specific content. But for freeform exploration, it is harder to keep the user's attention due to less content accessibility since the user needs to click and wait for the page to load before seeing new content. | Infinite scrolling is known to boost user engagement by enabling them to explore content without worrying about clicking on a button or link.                                                                                                                                                                                                             |
+| Navigation                       | Pagination enables users to quickly locate content, bookmark pages to be viewed later, and share page links with others, giving users more control of their browsing experience.                                                                                           | Locating content harder with infinite scrolling, Bookmarking a page or sharing it’s link is of little to no use because users will be taken back to the top of the page whenever the site is refreshed.                                                                                                                                                   |
+| Site footer                      | With pagination the footer can be placed right were it ought to be — at the bottom of the page.                                                                                                                                                                            | With infinite scrolling not appropriate to place a footer at the bottom of the page since it can be overwhelming for users to scroll to the very bottom of the page. <br>Some alternatives to this included placing the footer links in the sidebar or adding a load more button for users to click to fetch more content rather than fetching at scroll. |
```

Comment:
Please can you make this prose instead of a table?

Reply:
Okay. I will do that